.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseDatePeriodExtractor.cs

```diff
@@ -595,6 +613,18 @@ private List<Token> MergeMultipleExtractions(string text, List<ExtractResult> ex
                         idx += 2;
                         continue;
                     }
+
+                    // handle "between...and..." case when "between" follows the datepoints
+                    var afterStr = text.Substring(periodEnd, text.Length - periodEnd);
+                    if (this.config.GetBetweenTokenIndex(afterStr, out int afterIndex))
```

Comment:
Shouldn't this also check for the before/after flag? Or should it really run everytime?

Reply:
added check for before/after flag.